buildbot/master/files/config/factories.py

```diff
@@ -211,11 +212,12 @@ def make_step(self, command):
                 step_kwargs['logfiles'][logfile] = logfile
 
             # Provide environment variables for s3cmd
-            elif arg == './etc/ci/upload_nightly.sh':
+            elif arg == './etc/ci/upload_nightly.sh' or
+                 arg == r'.\etc\ci\upload_nightly.sh':
```

Comment:
I recommend surrounding this in brackets and using block ident:
```python
elif (
    arg == './etc/ci/upload_nightly.sh'
    or arg == '.\etc\ci\upload_nightly.sh'
):
    step_kwargs['logEnviron'] = ...
```
Another option that IMO looks nicer (although loses short-circuiting) is:
```python
elif any([
    arg == './etc/ci/upload_nightly.sh',
    arg == r'.\etc\ci\upload_nightly.sh',
]):
    step_kwargs['logEnviron'] = ...
```